target/linux/ramips/dts/E220.dts

```diff
@@ -0,0 +1,208 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "e220", "mediatek,mt7620a-soc";
+	model = "Maestro Wireless Solutions E220";
+
+    chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+        activity {
+            /* gpio27 */
+			label = "e220:orange:activity";
+			gpios = <&gpio1 3 GPIO_ACTIVE_HIGH>;
+		};
+        alert {
+            /* gpio29 */
+			label = "e220:red:alert";
+			gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+		};
+		network {
+            /* gpio26 */
+			label = "e220:orange:network";
+			gpios = <&gpio1 2 GPIO_ACTIVE_HIGH>;
+		};
+		power {
+            /* gpio60 */
+			label = "e220:green:power";
+			gpios = <&gpio2 20 GPIO_ACTIVE_HIGH>;
+        };
+		signal {
+            /* gpio28 */
+			label = "e220:orange:signal";
+			gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+		};
+		wifi {
+            /* w_lan */
+			label = "e220:blue:wifi";
+			gpios = <&gpio3 0 GPIO_ACTIVE_HIGH>;
+		};
+    };
+
+	gpio-keys {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+
+        input1 {
+            /* gpio30 */
+            label = "input1";
+			gpios = <&gpio1 6 GPIO_ACTIVE_HIGH>;
+        };
+        input2 {
+            /* gpio31 */
+            label = "input2";
+			gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
+        };
+        on-battery {
+            /* gpio33 */
+            label = "on-battery";
+			gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
+        };
+        on-poe {
+            /* gpio61 */
+            label = "on-poe";
+			gpios = <&gpio2 21 GPIO_ACTIVE_HIGH>;
+        };
+		reset {
+            /* gpio22  */
+			label = "reset";
+			gpios = <&gpio0 22 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+    };
+
+    gpio_export {
```

Comment:
Not sure which of these is really required. gpio-export is usually used to set gpios to a default value.

For example, on some boards the usb power needs to be enabled via gpio otherwise only external powered usb devices work. Since the usb driver has not support for enabling the usb power during load, it need to be done this way.

Please check if really all gpios listed need to be enabled/set here.